### ISSUE TITLE
Says gamertag does not exist when it does

### ISSUE THREAD

**Issue by andermoran** [flotwig/360api-gae#1]
It's more of an issue with gamercard.xbox.com I think but for some names it says it does not exist when it actually does. For example "abridge" has been taken but when it is put into the link it returns
"GamertagExists": false

**Comment by flotwig** [comment]
360api scrapes data from http://gamercard.xbox.com/ which does not distinguish between gamertags which do not exist and gamertags which are private.

See:  
http://gamercard.xbox.com/abridge.card  
http://gamercard.xbox.com/agamertagwhichdoesnotexist232545.card

**Comment by andermoran** [comment]
I don't believe it's possible for a gamer tag to be private. Perhaps it's the username to a Microsoft Zune account?
